HelpCentre/QA/flows/scheduled-topics-no-response-followup.md

```diff
@@ -0,0 +1,258 @@
+# Topics no response follow-up
+
+This is a scheduled flow that gets set to run 15 minutes after the AAQ FAQ Topics List is displayed, but we received no input from the user
+
+## Contact fields
+
+* `nav_bypass` This is a mechanism to bypass the main menu and send the user straight to a sub section
+* `aaq_metadata` This is used to keep track of the metadata from the latest AAQ call
+
+## Flow results
+
+* `aaq_faq_list_helpful`, Get set to `yes` or `no` depending on what the user clicks
+
+## Connections to other stacks
+
+* This is a scheduled reminder stack that gets set to run 15 minutes after the AAQ Topics List is shown, but we haven't received any input from the user
+
+## Global variables
+
+The following variable(s) are set in the `settings` global dictionary
+
+* `contentrepo_qa_token` used to auth api calls to CMS
+* `mc_ndoh_hub_token` used to authenticate calls to AAQ via MomConnect NDOH hub
+
+## Content dependencies
+
+* `mnch_onboarding_error_handling_button`
+* `plat_help_topics_no_response_follow_up`
+* `plat_help_acknowledgement_positive_`
+* `plat_help_acknowledgement_negative_`
+
+<!-- { section: "74f01d92-37ab-4c1e-8851-75f362fbde95", x: 0, y: 0} -->
+
+```stack
+card FetchError, then: TopicsNoResponseFollowup do
+  # Get AAQ Metadata for sending feedback, and potentially redirecting the user back to the AAQ Results page
+  aaq_metadata = parse_json("@contact.aaq_metadata")
+  inbound_id = aaq_metadata.inbound_id
+  feedback_secret_key = aaq_metadata.feedback_secret_key
+
+  # Fetch and store the error message, so that we don't need to do it for every error card
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_error_handling_button"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  # We get the page ID and construct the URL, instead of using the `detail_url` directly, because we need the URL parameter for `get` to start with `https://`, otherwise stacks gives us an error
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
```

Comment:
For new Journeys, can we start switching to making this a single HTTP call? i.e. adding the `?whatsapp=true` to the query string of the first call, then `button_error_text = search.body.results[0].body.text.value.message`